src/app/recipes/recipe-from/recipe-from.component.html

```diff
@@ -0,0 +1,291 @@
+<h2 class="mb-3">Recipe</h2>
+<form
+	[formGroup]="recipeForm"
+	#ngForm="ngForm"
+	(ngSubmit)="handleSubmit()"
+	*ngIf="!loading">
+	<div class="row">
+		<div class="col-12 col-md-9 col-xl-8">
+			<div class="title">
+				<label for="title" class="form-label fs-5 fw-bold">{{
+					'Recipes.Title' | translate
+				}}</label>
+				<input
+					type="text"
+					formControlName="title"
+					[placeholder]="'Recipes.Title' | translate"
+					class="form-control"
+					[class.is-invalid]="
+						(title?.touched || ngForm.submitted) && !!title?.errors
+					"
+					id="title" />
+				<div
+					class="erros"
+					*ngIf="(title?.touched || ngForm.submitted) && !!title?.errors">
```

Comment:
why not just !recipeForm.controls?.get('name')?.valid

Reply:
I don't wont to user see error on start. In my opinion error should be visible after try of submitting form or when input lose focus.

Reply:
ok, error shouldn't be visible in this case when user didn't touch input, but if it's not working it's ok